PROTOCOL.md

```diff
@@ -94,128 +240,815 @@ GET {prefix}/shares?maxResults=10&pageToken=...
 
-Example:
+<details open>
+<summary><b>200: The tables were successfully returned.</b></summary>
+
+<table>
+<tr>
+<th>HTTP Response</th>
+<th>Value</th>
+</tr>
+<tr>
+<td>Header</td>
+<td>
+
+`Content-Type: application/json; charset=utf-8`
+
+</td>
+</tr>
+<tr>
+<td>Body</td>
+<td>
+
+```json
+{
+  "items": [
+    {
+      "name": "string",
+      "schema": "string",
+      "share": "string",
+      "id": "string"
+    }
+  ],
+  "nextPageToken": "string",
+}
+```
 
+Note: the `items` field may be an empty array or missing when no results are found. The client must handle both cases.
+
+Note: the `id` field is optional. If `id` is populated for a table, its value should be unique across the sharing server and immutable through the table's lifecycle. 
+
+Note: the `nextPageToken` field may be an empty string or missing when there are no additional results. The client must handle both cases.
+</td>
+</tr>
+</table>
+</details>
+<details>
+<summary><b>400: The request is malformed.</b></summary>
+
+<table>
+<tr>
+<th>HTTP Response</th>
+<th>Value</th>
+</tr>
+<tr>
+<td>Header</td>
+<td>Content-Type: application/json</td>
+</tr>
+<tr>
+<td>Body</td>
+<td>
+
+```json
+{
+  "errorCode": "string",
+  "message": "string"
+}
 ```
-GET {prefix}/shares/vaccine_share/schemas/acme_vaccine_data/tables?maxResults=10&pageToken=...
 
+</td>
+</tr>
+</table>
+</details>
+<details>
+<summary><b>401: The request is unauthenticated. The bearer token is missing or incorrect.</b></summary>
+
+<table>
+<tr>
+<th>HTTP Response</th>
+<th>Value</th>
+</tr>
+<tr>
+<td>Header</td>
+<td>Content-Type: application/json</td>
+</tr>
+<tr>
+<td>Body</td>
+<td>
+
+```json
 {
-   "items" : [
-      {
-         "share" : "vaccine_share",
-         "schema" : "acme_vaccine_data",
-         "name" : "vaccine_ingredients",
-         "id": "dcb1e680-7da4-4041-9be8-88aff508d001"
-            
-      },
-      {
-         "share" : "vaccine_share",
-         "schema" : "acme_vaccine_data",
-         "name" : "vaccine_patients",
-         "id": "c48f3e19-2c29-4ea3-b6f7-3899e53338fa"
-      }
-   ],
-   "nextPageToken" : "..."
+  "errorCode": "string",
+  "message": "string"
+}
+```
+
+</td>
+</tr>
+</table>
+</details>
+<details>
+<summary><b>403: The request is forbidden from being fulfilled.</b></summary>
+
+<table>
+<tr>
+<th>HTTP Response</th>
+<th>Value</th>
+</tr>
+<tr>
+<td>Header</td>
+<td>Content-Type: application/json</td>
+</tr>
+<tr>
+<td>Body</td>
+<td>
+
+```json
+{
+  "errorCode": "string",
+  "message": "string"
+}
+```
+
+</td>
+</tr>
+</table>
+</details>
+<details>
+<details>
```

Comment:
These `details` tag are not added properly If you check the preview, it folded everything below.

Reply:
Whoops. Thanks!